apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -356,6 +362,58 @@ stop_node(N, Config) ->
 create_channel(Cfg) ->
     with_trace(fun t_create_channel_/1, Cfg, "create_channel").
 
+multiple_responder_keys_per_port(Cfg) ->
+    Slogan = ?SLOGAN,
+    %% Debug = get_debug(Cfg),
+    Debug = true,
```

Comment:
This was maybe meant to rather be deleted, and the line above uncommented?

Reply:
Yes, though it might be good to keep the debug on for now. :)